src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -3,10 +3,26 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
+import platform
+
 from azure.cli.core.commands import register_cli_argument
 from azure.cli.core.commands.parameters import (
     name_type,
     resource_group_name_type)
+from azure.cli.core._util import CLIError
```

Comment:
After making my other suggested change, this import would be unused so can be removed.